src/appleseed/renderer/modeling/aov/depthaov.cpp

```diff
@@ -67,8 +67,8 @@ namespace
       : public UnfilteredAOVAccumulator
     {
       public:
-        DepthAOVAccumulator(Image& image, Image& filter_image)
-          : UnfilteredAOVAccumulator(image, filter_image)
+        DepthAOVAccumulator(Image& image)
```

Comment:
Make explicit (fix everywhere).

All single-argument constructors should be marked as `explicit`, unless implicit conversions are actually desired. In fact, in the future we might even make all multiple-arguments constructors explicit as well since this is relevant in C++11 and beyond.